cost-analyzer/templates/aggregator-statefulset.yaml

```diff
@@ -0,0 +1,189 @@
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostAggregator) }}
+{{- if gt (.Values.kubecostAggregator.replicas | toString | atoi) 0 }}
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "aggregator.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "aggregator.commonLabels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.kubecostAggregator.replicas }}
+  serviceName: "aggregator"
```

Comment:
This value does not use the templated value.

Reply:
fixed